src/test/java/com/app/persistent/JsonParserTest.java

```diff
@@ -20,4 +21,43 @@ void writeTest() throws FileNotFoundException {
 
         JsonParser.write("config", parameters);
     }
+
+    @Test
+    void writeTestWithValues() throws FileNotFoundException {
+        EnvironmentParameters parameters = new EnvironmentParameters();
+
+        DisplayYY displayYY = new DisplayYY();
+        displayYY.setX(FREQUENCY);
```

Comment:
enum sa pouziva skor takto: MeasuredQuantity.FREQUENCY, neroli sa import, nie je to tressa